spring-webmvc/src/test/java/org/springframework/web/servlet/mvc/method/annotation/CrossOriginTests.java

```diff
@@ -345,6 +376,34 @@ public void baz() {
 
 	}
 
+	@Target({ElementType.METHOD, ElementType.TYPE})
+	@Retention(RetentionPolicy.RUNTIME)
+	@Documented
+	@CrossOrigin
+	private @interface CrossOriginMetaAnnotation {
```

Comment:
> @CrossOrigin is a meta-annotation in this context.

This is the meaning of the annotation name. Is `CrossOriginSetAsMetaAnnotation` better ?